### DISCUSSION TITLE
Guide me to be a mobile developer.

### DISCUSSION THREAD

**Discussion by richbeta312** [community/community#139509] [Mobile] [answered]
### Select Topic Area

General

### Body

Recently, my client noticed me that they need the mobile application but I don't know how to kick start the project.
Please guide me if you are open-minded.

**Comment by Oleksii9999** [comment, marked as answer]
hi, my friend william has deep understanding mobile applications. he is expert in flutter. please let me know if you are interested. thanks

**Comment by arslan578** [comment]
To kickstart the mobile application project, start by gathering clear requirements from your client about the app’s features and functionality. Then, choose a mobile development framework like React Native or Flutter, set up your development environment, and begin with simple components to build the app.
